lerobot/common/datasets/factory.py

```diff
@@ -71,17 +72,36 @@ def make_dataset(cfg, split: str = "train") -> LeRobotDataset | MultiLeRobotData
 
     resolve_delta_timestamps(cfg)
 
-    # TODO(rcadene): add data augmentations
+    if cfg.image_transform.enable:
+        transform = v2.Compose(
+            [
+                v2.ColorJitter(
+                    brightness=cfg.image_transform.colorjitter_factor,
+                    contrast=cfg.image_transform.colorjitter_factor,
+                ),
+                v2.RandomAdjustSharpness(
+                    cfg.image_transform.sharpness_factor, p=cfg.image_transform.sharpness_p
+                ),
+                v2.RandomAdjustSharpness(cfg.image_transform.blur_factor, p=cfg.image_transform.blur_p),
```

Comment:
Typo?
```suggestion
                v2.RandomAdjustBlur(cfg.image_transform.blur_factor, p=cfg.image_transform.blur_p),
```
(I dont know if RandomAdjustBlur exists)

Reply:
If using RandomAdjustSharpness with a parameter < 1, it will blur the image (Using the transform with parameter = 1 does not change the image). And we use two different parameters for implementing blur and sharpness. I think it is the simplest way to create blur 
There is also the option for GaussianBlur, it requires parameters for the kernel size and the variance of the gaussian.

Reply:
Sounds good. We need to add a comment since it's not obvious ^^ People like me would think it's a typo.